angular.json

```diff
@@ -32,30 +32,36 @@
             ],
             "styles": [
               "src/main.scss",
-              {"inject":false,
+              {
```

Comment:
Remove changes to this file.

Reply:
The motive of your pr is not to edit these files. It would be better to have such changes in separate pr.